driver-core/src/main/com/mongodb/MongoCredential.java

```diff
@@ -179,6 +182,17 @@ public final class MongoCredential {
     @Beta(Beta.Reason.CLIENT)
     public static final String AWS_CREDENTIAL_PROVIDER_KEY = "AWS_CREDENTIAL_PROVIDER";
 
+    public static final String PROVIDER_NAME = "PROVIDER_NAME";
+
+    public static final String REQUEST_TOKEN_CALLBACK = "REQUEST_TOKEN_CALLBACK";
```

Comment:
Naming? Prefix? Maybe "TOKEN" should not be present, perhaps "OIDC_ON_REQUEST" or "OIDC_REQUEST_CALLBACK"? Spec change.

Reply:
These will not be renamed, since this has shipped in other drivers.